mlflow/environment_variables.py

```diff
@@ -110,6 +110,10 @@ def get(self):
 #: See https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html for more information.
 MLFLOW_TRACKING_AWS_SIGV4 = _BooleanEnvironmentVariable("MLFLOW_TRACKING_AWS_SIGV4", False)
 
+#: Specifies the auth for MLFlow HTTP request
+#: (default: ``None``). When set, it will overwrite the "Authorization" HTTP header.
+MLFLOW_TRACKING_AUTH = _EnvironmentVariable("MLFLOW_TRACKING_AUTH", str, None)
```

Comment:
Does this environment variable specify a provider name?

Reply:
Yes, the `MLFLOW_TRACKING_AUTH` specify the provider name with a default value `None`

Reply:
Got it, can we update the comment to be more accurate?

Reply:
Just updated